custom/icds_reports/reports/adhaar.py

```diff
@@ -88,9 +88,6 @@ def get_data_for(filters):
 
 def get_adhaar_sector_data(domain, config, loc_level, show_test=False):
     group_by = ['%s_name' % loc_level]
-    if loc_level == LocationTypes.SUPERVISOR:
```

Comment:
Why are you removing these?

Reply:
because previously we show several bars in sector view and we had to calculate this by the location below selected location now when we show only one bar we can get value from selected location.